### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by srid** [gnome-integration-team/chrome-gnome#8]
No commits in the last 7.5 years, and PRs have been left languishing.

The theme on Chrome Webstore certainly looks visibly different to the current Gnome theme adwaita.

**Comment by garrett** [comment]
I'd love to be proven wrong, but... it's not really possible to theme Chrome. The most one can do is change the colors a bit and use images as backgrounds. There's really no way forward to make it look like GNOME.

The most that can be done is to update the colors that are used, as GNOME & GTK has made subtle changes to the colors over the years.

**Comment by garrett** [comment]
I guess a good first step would be to merge https://github.com/gnome-integration-team/chrome-gnome/pull/7 and build on top of that.

**Comment by garrett** [comment]
Meanwhile, it looks like the next stable release of GNOME ([version 40](https://discourse.gnome.org/t/new-gnome-versioning-scheme/)) will drop the blue lines, so simply using the stock GTK theme baked into Chrome under its appearance settings would be the closest one could get to GNOME's look with Chrome when that comes out.

The most recent tab mockup (from a month ago) has line-less tabs, for example:
![tabs mockup](https://gitlab.gnome.org/Teams/Design/os-mockups/-/raw/master/tabs/tabs.png)

GNOME Web (Epiphany) currently has a branch with reworked tabs that match the new style: https://gitlab.gnome.org/GNOME/epiphany/-/merge_requests/814. (There are even flatpaks available to test, too — and there are screenshots of the tabs in the issue, of course.)

![gnome web](https://gitlab.gnome.org/GNOME/epiphany/uploads/69c10fe971217c904f428e7fb434f8b4/Screenshot_from_2020-09-06_21-34-01.png)

In summary: the one thing a Chrome theme could add (a faked blue line) is going to be gone soon anyway.

The best way to have a browser that completely matches GNOME is to use GNOME Web (which is fantastic in 3.38) or Firefox with [a userCSS based theme](https://github.com/rafaelmardojai/firefox-gnome-theme). Chrome is simply not flexible enough, as it only lets one set colors and background images.
